### ISSUE TITLE
Turn plone/meta into a real installable Python package

### ISSUE THREAD

**Issue by mauritsvanrees** [plone/meta#239]
This was done for Zope meta, so it could be nice for us as well.  See https://github.com/zopefoundation/meta/pull/282

Theoretically we could even depend on the new `zope.meta` package so more code is shared.

But I am not volunteering to do the work.  I have other fish to fry. :-)
So: just another idea.

**Comment by gforcada** [comment]
Yes, I've thought about that option, but also thinking of other solutions:

- a bot that reacts on github repositories (i.e. dependabot-like)
- as an action that can be scheduled triggered at each repository's convenience

But actually, if we create it as a installable package, we can still do the above as well 👍🏾 

I would volunteer, but given how life is pushing me around as of late, I don't expect to do it soonish 🫠 

The reference to the zope.meta is great though, so we can see how they did it 😄
